examples/app_commands/info.py

```diff
@@ -1,45 +1,43 @@
-import discord
-from discord.ext import commands
+# This example requires the 'members' privileged intent to use the Member converter.
 
-# imports
+import discord
 
-intents = discord.Intents(
-    guilds=True,
-    members=True,
-    messages=True,
-)
+intents = discord.Intents.default()
+intents.members = True
 
-bot = commands.Bot(
-    command_prefix="/",
-    description="An example to showcase how to extract info about users",
+bot = discord.Bot(
+    debug_guilds=[...],
+    description="An example to showcase how to extract info about users.",
     intents=intents,
 )
 
 
-@bot.slash_command(name="userinfo", description="gets the info of a user")
+@bot.slash_command(name="userinfo", description="Gets info about a user.")
 async def info(ctx, user: discord.Member = None):
-    user = user or ctx.author  # if no user is provided it'll use the the author of the message
-    e = discord.Embed()
-    e.set_author(name=user.name)
-    e.add_field(name="ID", value=user.id, inline=False)  # user ID
-    e.add_field(
-        name="Joined",
-        value=discord.utils.format_dt(user.joined_at, "F"),
-        inline=False,
-    )  # When the user joined the server
-    e.add_field(
+    user = user or ctx.author  # If no user is provided it'll use the author of the message
+    embed = discord.Embed()
+    embed.set_author(name=user.name)
+    embed.set_thumbnail(url=user.display_avatar.url)
+    embed.add_field(name="ID", value=user.id, inline=False)  # user ID
+    embed.add_field(
         name="Created",
         value=discord.utils.format_dt(user.created_at, "F"),
         inline=False,
     )  # When the user's account was created
-    colour = user.colour
-    if colour.value:  # if user has a role with a color
-        e.colour = colour
 
-    if isinstance(user, discord.User):  # checks if the user in the server
-        e.set_footer(text="This member is not in this server.")
+    if user.colour.value:  # If user has a role with a color
+        embed.colour = user.colour
+
+    if isinstance(user, discord.User):  # Checks if the user in the server
+        embed.set_footer(text="This user is not in this server.")
+    else:  # We end up here if the user is a discord.Member object
+        embed.add_field(
+            name="Joined",
+            value=discord.utils.format_dt(user.joined_at, "F"),
+            inline=False,
+        )  # When the user joined the server
 
-    await ctx.respond(embed=e)  # sends the embed
+    await ctx.respond(embeds=[embed])  # Sends the embed
```

Comment:
This doesn’t need to be changed, mostly because `embed` doesn’t use the deprecated embed param, the last time I saw.

Reply:
Changed it purely for readability purposes, can revert it to the old syntax if that'd be preferred.

Reply:
Discord itself only accepts a list of embeds, so our examples should follow that practice. It also makes the code more pythonic by explicitly passing a single-item list instead of having the library implicitly handle the conversion. 

I would keep the current method of passing the list of embeds.